rmgpy/rmg/model.py

```diff
@@ -1199,8 +1199,12 @@ def prune(self, reactionSystems, toleranceKeepInEdge, maximumEdgeSpecies, minSpe
                 pruneDueToRateCounter += 1
             # Keep removing species with the lowest rates until we are below the maximum edge species size
             elif numPrunableSpecies - len(speciesToPrune) > maximumEdgeSpecies:
-                logging.info('Pruning species {0} to make numEdgeSpecies smaller than maximumEdgeSpecies'.format(spec)) # repeated ~15 lines below
-                speciesToPrune.append((index, spec))
+                if not maxEdgeSpeciesRateRatios[index] < toleranceMoveToCore:
+                    logging.info('Pruning species {0} to make numEdgeSpecies smaller than maximumEdgeSpecies'.format(spec)) # repeated ~15 lines below
```

Comment:
leaving `~15 lines below` isn't helpful if this section is changed again. Can you give a more informative comment, or just say `below`?

Reply:
I didn't add the repeated ~15 lines below and I'm not really sure what it refers to.

Reply:
I've removed that comment now.